src/lib/accessibility/FocusZone/FocusUtilities.ts

```diff
@@ -0,0 +1,404 @@
+/* tslint:disable:no-string-literal */
+
+const IS_FOCUSABLE_ATTRIBUTE = 'data-is-focusable'
```

Comment:
Since we removed utilities lib, let's then remove ```IS_FOCUSABLE_ATTRIBUTE``` from ```interfaces.ts```
and export these attributes from here to use them in behaviors